<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>29</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="btnActionEdit">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Edit</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background: none;
border: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../assets/edit-green.png</normaloff>../assets/edit-green.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnActionDelete">
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Delete</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background: none;
border: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../assets/trash-red.png</normaloff>../assets/trash-red.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnActionDone">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>24</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Done</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border: 1px solid rgb(26, 194, 161);
border-radius: 5px;
	color: rgb(26, 194, 161);
}
QPushButton:hover, QPushButton:focus{
	background-color: rgb(26, 194, 161);
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
